config/locales/devise.es.yml

```diff
@@ -41,6 +59,12 @@ es:
       updated: 'Actualizaste tu cuenta correctamente.'
       update_needs_confirmation: "Actualizaste tu cuenta correctamente, pero tenemos que revalidar tu email. Revisa tu correo para confirmar la dirección."
       destroyed: 'Adiós, tu cuenta ha sido eliminada. Esperamos verte de vuelta pronto!'
+      new:
+        title: "Registrarse"
+        password_min_txt: "(mínimo %{caracteres} caracteres)"
+        terms_txt: "He leído y acepto los términos y condiciones"
+        data_policies_txt: "Políticas de protección de datos personales"
```

Comment:
yo sé que lo pasaste como estaba, pero me molesta un poco que en inglés diga "I have read and accept the personal data policy" pero en español solo diga "Políticas de protección de datos personales", creo que debería ser como los tŕeminos y condiciones y colocar el texto completo "he leído..."